### PR TITLE
Switch to scalafmt with config same as newer akka projects

### PR DESCRIPTION
**Pull request by chbatey**
I don't feel strongly about the changes I did this to confirm with other new projects in our ecosystem and IntelliJ supports scalafmt out of the box

### CODE REVIEW

**Comment by ignasi35** [comment]
LGTM

File: core/src/main/scala/akka/persistence/cassandra/CassandraPluginConfig.scala [hunk 1]
```diff
@@ -57,26 +60,36 @@ object CassandraPluginConfig {
...
 
     def getDataCenterReplicationFactorList(dcrfList: Seq[String]): String = {
       val result: Seq[String] = dcrfList match {
-        case null | Nil => throw new IllegalArgumentException("data-center-replication-factors cannot be empty when using NetworkTopologyStrategy.")
-        case dcrfs => dcrfs.map {
-          dataCenterWithReplicationFactor =>
+        case null | Nil =>
+          throw new IllegalArgumentException(
+            "data-center-replication-factors cannot be empty when using NetworkTopologyStrategy."
+          )
```
Comment (patriknw) [inline core/src/main/scala/akka/persistence/cassandra/CassandraPluginConfig.scala]: that `)` is weird

File: core/src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala [hunk 2]
```diff
@@ -186,27 +168,31 @@ class ConfigSessionProvider(system: ActorSystem, config: Config) extends Session
    *
    * @param clusterId the configured `cluster-id` to lookup
    */
-  def lookupContactPoints(clusterId: String)(implicit ec: ExecutionContext): Future[immutable.Seq[InetSocketAddress]] = {
+  def lookupContactPoints(
+      clusterId: String
```
Comment (patriknw) [inline core/src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala]: hm

File: core/src/main/scala/akka/persistence/cassandra/journal/CassandraStatements.scala [hunk 3]
```diff
@@ -229,7 +229,7 @@ trait CassandraStatements {
         sequence_nr = ?
     """
 
-  private[akka] def deleteMessages = {
+  private[akka] def deleteMessages =
```
Comment (patriknw) [inline core/src/main/scala/akka/persistence/cassandra/journal/CassandraStatements.scala]: personally I like the `{` when the single statement is big chunk of code

Comment (chbatey) [reply core/src/main/scala/akka/persistence/cassandra/journal/CassandraStatements.scala]: I will remove the `RedundentBraces` so we can leave them in as and when we like them

File: core/src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala (hunk 2 above)
Comment (chbatey) [reply core/src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala]: Not a fan? It broke the line due to length, i think just moving the second parameter list to the second line is how i'd have done it manually

File: core/src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala [hunk 4]
```diff
@@ -186,27 +168,31 @@ class ConfigSessionProvider(system: ActorSystem, config: Config) extends Session
...
+        hosts map { ipWithPort =>
           ipWithPort.split(":") match {
             case Array(host, port) => new InetSocketAddress(host, port.toInt)
-            case Array(host)       => new InetSocketAddress(host, port)
-            case msg               => throw new IllegalArgumentException(s"A contact point should have the form [host:port] or [host] but was: $msg.")
+            case Array(host) => new InetSocketAddress(host, port)
+            case msg =>
+              throw new IllegalArgumentException(
+                s"A contact point should have the form [host:port] or [host] but was: $msg."
+              )
```
Comment (ignasi35) [inline core/src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala]: I think I like the `=>` alignment better but I'm fine with the new format too.

**Pull request merged**